client/db/test/dbtest.go

```diff
@@ -36,7 +37,7 @@ func RandomWallet() *db.Wallet {
 		Account:   ordertest.RandomAddress(),
 		INIPath:   ordertest.RandomAddress(),
 		Balance:   rand.Uint64(),
-		BalUpdate: time.Unix(int64(rand.Uint64()), 0),
+		BalUpdate: encode.UnixTimeMilli(encode.UnixMilli(time.Now())),
```

Comment:
It's a bit roundabout.  We have simply been doing `time.Now().Truncate(time.Milliseconds).UTC()`.  In spots you will find this function called `nowMs` or similar.  I suppose this could be made a new convenience function in `encode` or `dex`.